rocksdb/tests/test_options.py

```diff
@@ -165,7 +202,88 @@ def test_compaction_opts_universal(self):
+                ('skip_log_error_on_recovery', False, True),
+                ('stats_dump_period_sec', 600, 3600),
+                ('stats_persist_period_sec', 600, 3600),
+                ('persist_stats_to_disk', False, True),
+                ('stats_history_buffer_size', 1024*1024, 1024),
+                ('advise_random_on_open', True, False),
+                ('db_write_buffer_size', 0, 100),
+                ('new_table_reader_for_compaction_inputs', False, True),
+                ('compaction_readahead_size', 0, 10),
+                ('random_access_max_buffer_size', 1024*1024, 100),
+                ('writable_file_max_buffer_size', 1024*1024, 100),
+                ('use_adaptive_mutex', False, True),
+                ('bytes_per_sync', 0, 10),
+                ('wal_bytes_per_sync', 0, 10),
+                ('strict_bytes_per_sync', False, True),
+                ('enable_thread_tracking', False, True),
+                ('delayed_write_rate', 0, 10),
+                ('enable_pipelined_write', False, True),
+                ('unordered_write', False, True),
+                ('allow_concurrent_memtable_write', True, False),
+                ('enable_write_thread_adaptive_yield', True, False),
+                ('max_write_batch_group_size_bytes', 1 << 20, 10),
+                ('write_thread_max_yield_usec', 100, 200),
+                ('write_thread_slow_yield_usec', 3, 2000),
+                ('skip_stats_update_on_db_open', False, True),
+                ('skip_checking_sst_file_sizes_on_db_open', False, True),
+                ('allow_2pc', False, True),
+                ('fail_if_options_file_error', False, True),
+                ('dump_malloc_stats', False, True),
+                ('avoid_flush_during_recovery', False, True),
+                ('avoid_flush_during_shutdown', False, True),
+                ('allow_ingest_behind', False, True),
+                ('preserve_deletes', False, True),
+                ('two_write_queues', False, True),
+                ('manual_wal_flush', False, True),
+                ('atomic_flush', False, True),
+                ('avoid_unnecessary_blocking_io', False, True),
+                ('write_dbid_to_manifest', False, True),
+                ('log_readahead_size', 0, 10),
+                ('best_efforts_recovery', False, True),
+        ):
+            with self.subTest(option=option):
+                opts = rocksdb.Options()
+                if def_value is NOTNONE:
+                    self.assertIsNotNone(getattr(opts, option))
+                else:
+                    self.assertEqual(def_value, getattr(opts, option))
+                if new_value is UNSETTABLE:
+                    self.assertRaises(
+                        Exception, setattr, opts, option, new_value)
+                else:
+                    setattr(opts, option, new_value)
+                    self.assertEqual(getattr(opts, option), new_value)
+    def test_row_cache_set(self):
```

Comment:
missing empty line between methods

Reply:
In fact, this test can be folded up into the `test_rockdb_options` function since row_cache can be set using the object LRUCache. I've fixed that in the latest push.